farm/infer.py

```diff
@@ -251,6 +251,13 @@ def inference_from_dicts(self, dicts, rest_api_schema=False, max_processes=128):
                 f"b) ... run inference on a downstream task: make sure your model path {self.name} contains a saved prediction head"
             )
 
+        # Assign a sequential doc_id to each dictionary
+        old_dicts = dicts
```

Comment:
Maybe I am missing something here, but why do you need the "old_dict" var here? Can't we modify the dicts in place?

Reply:
You're right. Change made.